src/tracing/types.rs

```diff
@@ -114,6 +138,35 @@ impl CallTrace {
     }
 }
 
+/// Additional decoded data enhancing the [CallLog].
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+
+pub struct DecodedCallLog {
+    /// The decoded event name.
+    pub name: Option<String>,
+    /// The decoded log parameters, a vector of the parameter name (e.g. topic0) and the parameter
+    /// value (e.g. 0x9d3...45ca)
+    pub params: Option<Vec<(String, String)>>,
```

Comment:
I've updated to comment from `topic0` to generic `foo` as it is expected to be a named parameter as in

```
    ├─ [2696] Counter::log2()
    │   ├─ emit Log2(foo: 1, bar: 123, dump: 0x68692066726f6d206c6f6732)
    │   └─ ← [Stop] 
```

I think this can be part of the follow-up stylistic update of https://github.com/foundry-rs/foundry/pull/8207